pkg/testidentification/ocp_variants.go

```diff
@@ -49,7 +49,8 @@ var (
 	ovirtRegex     = regexp.MustCompile(`(?i)-ovirt`)
 	ovnRegex       = regexp.MustCompile(`(?i)-ovn`)
 	// proxy jobs do not have a trailing -version segment
-	ppc64leRegex      = regexp.MustCompile(`(?i)-ppc64le`)
```

Comment:
Its better to only add a new variant for powervs rather than updating the existing to `libvirt-ppc64le`.

Reply:
I would have thought making the other one more precise would be preferable, but I will go back to the more confusing way.